### PR TITLE
add PepQuery tool

### PR DESCRIPTION
**Pull request by jraysajulga**
Hoping to add the wrapper for PepQuery from https://pepquery.org.

Still undergoing testing...

### CODE REVIEW

File: tools/pepquery/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+categories: 
+  - Proteomics
+description: "A peptide-centric search engine for novel peptide identification and validation."
+name: pepquery
+owner: rsajulga
```
Comment (bgruening) [inline tools/pepquery/.shed.yml]: galaxyp

File: tools/pepquery/pepquery.xml [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+    <description>Peptide-centric search engine for novel peptide identification and validation.</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <requirements> 
+        <requirement type="package" version="@VERSION@">pepquery</requirement>
+    </requirements>
+    <command detect_errors="exit_code">
+        <![CDATA[
+            cp $req_inputs.spectrum_file ${req_inputs.spectrum_file}.mgf &&
```
Comment (bgruening) [inline tools/pepquery/pepquery.xml]: This means you copy the file to the Galaxy database, thats not a good idea. Pleaes copy, or better link them to the current working dir, e.g. `./`

File: tools/pepquery/pepquery.xml [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+                <expand macro="modifications" />
+             </param>
+             <param name="var_mod" label="Variable modification(s)" type="select" multiple="true" argument="-varMod" help="Variable modification" >
+                <option value="107">Oxidation of M (15.99491461956)[0]</option>
+                <option value="6" selected="true">Carbamidomethylation of C (57.02146372057)[0]</option>
+                <expand macro="modifications" />
+             </param>
+             <param label="Max Modifications" name="max_mods" type="integer" value="3" argument="-maxVar" help="Max number of variable modifications" />
+             <param label="Unmodified?" name="unmodified" type="boolean" value="false" argument="-um" help="Validation with unrestricted modification searching" />
+             <param label="Consider amino acid substitution modifications?" name="aa" type="boolean" value="false" argument="-aa" help="Whether or not to consider aa substitution modifications when perform modification filtering." />
```
Comment (bgruening) [inline tools/pepquery/pepquery.xml]: booleans only have truevalue and falsevalue.

If you set `truevalue="-aa"` and `falsevalue=""` than your cheetah becomes simply `$modifications.aa`. Which evaluates to `-aa` in the true case and ` ` in the false case :)

Comment (jraysajulga) [reply tools/pepquery/pepquery.xml]: Ah. I like that solution. Thanks.

**Comment by bernt-matthias** [comment]
superseded by https://github.com/galaxyproteomics/tools-galaxyp/pull/417 ?

**Comment by jraysajulga** [comment]
Correct. Closing this now.

**Pull request closed without merging**